pkgs/applications/version-management/git-and-tools/git-annex-remote-googledrive/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, annexremote
+, drivelib
+, GitPython
+, tenacity
+, humanfriendly
+}:
+
+buildPythonApplication rec {
+  pname = "git-annex-remote-googledrive";
+  version = "1.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1031scjwd5mc2igayg4rbmvb89xdxf1fwkw2101kq3p0grvikp0c";
+  };
+
+  propagatedBuildInputs = [ annexremote drivelib GitPython tenacity humanfriendly ];
+
+  # no tests
```

Comment:
```suggestion
  # package does not contain tests
```

Reply:
Changed it to the most common form:
```
$ rg 'doCheck = false' -C1 | rg '#.*$' | sed 's/.*\(#.*$\)/\1/' | sort | uniq -c | sort -h | tail
     11 # no tests implemented
     12 # tests not included with pypi release
     13 # no tests are present
     15 # error: invalid command 'test'
     18 # no tests included
     21 # No tests included
     25 # no tests
     30 # No tests
     46 # No tests in archive
     80 # has no tests
```

Reply:
You can do sort -u.

Reply:
No, `sort -u` merely removes duplicates, it does not count the number of occurrences.